src/components/OptionsList/BaseOptionsList.js

```diff
@@ -158,11 +237,12 @@ class BaseOptionsList extends Component {
                     onScrollBeginDrag={this.props.onScrollBeginDrag}
                     contentContainerStyle={this.props.contentContainerStyles}
                     showsVerticalScrollIndicator={false}
-                    sections={_.filter(this.props.sections, section => section.data.length !== 0)}
+                    sections={this.props.sections}
```

Comment:
Having this discrepancy made inconsistencies between `this.props.sections` and the `data` passed to `getItemLayout`. AFAICT there's no reason we needed to filter out empty sections so this is simpler and allows us to compute layouts up-front without having to filter the sections first.